ArticleTemplates/assets/js/modules/creativeInjector.js

```diff
@@ -13,13 +13,27 @@ function (
         if (!initialised) {
             initialised = true;
 
-            window.injectCreative = injectCreative;
-            window.applyNativeFunctionCall('injectInlineCreative');
+            if (GU.opts.contentType === 'liveblog' || document.querySelector('.article__body--liveblog')) {
+              trackLiveBlogEpic();
+            } else {
+              window.injectCreative = injectCreative;
+              window.applyNativeFunctionCall('injectInlineCreative');
+            }
+        }
+    }
+
+    function trackLiveBlogEpic() {
+        var liveBlogEpicContainer = document.querySelector('contributions-epic');
+
+        if (liveBlogEpicContainer) {
+            var liveBlogEpicContainerId = liveBlogEpicContainer.getAttribute('id')
+
+            isCreativeInView(liveBlogEpicContainer, liveBlogEpicContainerId)
```

Comment:
Ah yes sorry, I was saying put the call to _window.addEventListener('scroll', util.debounce(isCreativeInView.bind(null, liveBlogEpicContainer, liveBlogEpicContainerId), 100))_ in a function. Then call this function from line 31.

You could then replace line 56 with a call to this new function as well.